src/DeviceInput/InputDevices/deviceSourceManager.ts

```diff
@@ -25,7 +26,7 @@ export class DeviceSourceManager {
     public readonly onDeviceDisconnectedObservable: Observable<DeviceSource<DeviceType>>;
 
     // Private Members
-    private _deviceSourceManager: InternalDeviceSourceManager;
+    private _deviceSourceManager: Nullable<InternalDeviceSourceManager>;
```

Comment:
This isn't necessary. You can get away with it without making this nullable.

Reply:
Removed Nullable